src/Storage/Database/Schema/Table/Tokens.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Bolt\Storage\Database\Schema\Table;
+
+/**
+ * Table for tokens data.
+ *
+ * @author Carlos Pérez <mrcarlosdev@gmail.com>
+ */
+class Tokens extends BaseTable
```

Comment:
If we do agree to a separate table, I would suggest that "Token" is too generic and we'd be better off with something like "Invitations"